role_scripts/15/standby/run.sh

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env bash
```

Comment:
update this file with latest config file by postgres 15 https://github.com/postgres/postgres/blob/REL_15_STABLE/src/backend/utils/misc/postgresql.conf.sample